src/service/resources-interface.js

```diff
@@ -178,12 +172,6 @@ export class ResourcesInterface {
    */
   updateLayoutPriority(element, newLayoutPriority) {}
 
-  /**
-   * https://github.com/ampproject/amphtml/issues/25428
-   * @return {boolean}
-   */
-  isIntersectionExperimentOn() {}
```

Comment:
I don't think we can clean this experiment up until we polyfill `InOb(document)` API.

Reply:
To clarify: the current polyfill does not currently support `new IntersectionObserver({root: document})`. This is not only IE11, but I think most of the browser versions > 1 year old.

Reply:
Oh no! I'll look into if we can get this prioritized

Reply:
I don't mean to scare anyone off, of course. I think the polyfill changes will be pretty modest. The only ugly part would be to override the native implementation and force the polyfill in case of an iframe. I think it might make sense to just go ahead and do this.

Reply:
> override the native implementation and force the polyfill in case of an iframe

We can solve this via try/catch right?

Reply:
Correct. Right now we do:

```
if (window.IntersectionObserver === undefined) {
  installPolyfill();
}
```

And instead we'll be doing:
```
if (window.IntersectionObserver === undefined) {
  installPolyfill();
} else if (isFramed(window)) {
  try {
    new window.IntersectionObserver(()=> {}, {root: document});
  } catch () {
    installPolyfill();
  }
}
```